docs/src/stories/components/Layout/Stack.stories.jsx

```diff
@@ -0,0 +1,464 @@
+      control: {
+        type: 'inline-radio',
+      },
+      description: 'Sets how items will be distributed in the stacking direction.', 
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'start',
+        },
+      },
+    },
+
+    narrow_spread: {
+      options: ['inherit', 'start', 'center', 'end', 'distribute', 'distributeEvenly'],
+      control: {
+        type: 'inline-radio',
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Wrap
+
+    wrap: {
+      options: ['wrap', 'nowrap'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Sets whether items are forced onto one line or can wrap onto multiple lines.',
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'nowrap',
+        }
+      }
+    },
+    narrow_wrap: {
+      options: ['inherit', 'wrap', 'nowrap'],
+      control: {
+        type: 'inline-radio'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Divider
+
+    showDividers: {
+      control: {
+        type: 'boolean'
+      },
+      description: `Whether a divider between items is shown or not.
+
+_Note: the presence of a divider duplicates the \`gap\` between items._`,
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'false',
+        }
+      }
+    },
+    dividerAriaRole: {
+      options: ['presentation', 'separator', 'none'],
+      control: {
+        type: 'inline-radio'
+      },
+      description: 'Sets which ARIA role will be used for the divider.',
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'presentation',
+        }
+      }
+    },
+    narrow_showDividers: {
+      control: {
+        type: 'boolean'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Children
+    children: {
+      description: 'A slot for children elements.',
+      table: {
+        category: 'HTML'
+      }
+    }
+  },
+};
+
+export const StackTemplate = ({
+  _debug,
+  _height,
+  _width,
+  direction,
+  gap,
+  gap_custom,
+  align,
+  alignWrap,
+  spread,
+  wrap,
+  showDividers,
+  dividerAriaRole,
+
+  narrow_direction,
+  narrow_gap,
+  narrow_gap_custom,
+  narrow_align,
+  narrow_alignWrap,
+  narrow_spread,
+  narrow_wrap,
+  narrow_showDividers,
+
+  children
+}) => {
+
+  let custom_styles  = {};
+
+  // Default values
+  direction = direction ?? 'block';
+  gap = gap ?? 'normal';
+  alignWrap = alignWrap ?? 'start';
+  dividerAriaRole = dividerAriaRole ?? 'presentation';
+
+  // Default narrow values
+  narrow_direction = narrow_direction ?? 'inherit';
```

Comment:
Non-blocking nitpick: You don't need to mutate the props, you could just set defaults when you spread the props argument.

For example:
```tsx
export const StackTemplate = ({
  direction = 'block'
  gap = 'normal'
  // ...etc
}) =>
```